### PR TITLE
New .pot, minified assets, changelog, and version bump

### PR DESCRIPTION
**Pull request by cklosowski**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -187,6 +187,43 @@ Yes, through the addition of one or more of the add-on payment gateways, you can
 
 == Changelog ==
 
+= 2.6.11, November 8, 2016 =
+* Fix: Correct translation for login shortcode redirect description.
+* Fix: Improve the caching of stats transients for better performance.
+* Fix: API Keys not able to be copied from user profile.
+* Fix: Correct link to the iOS application.
+* Fix: Use new iPhone icon.
+* Fix: EDD_Payment::setup_tax not always contain proper data.
```
Comment (chriscct7) [inline readme.txt]: did or does before the not

File: readme.txt [hunk 2]
```diff
@@ -187,6 +187,43 @@ Yes, through the addition of one or more of the add-on payment gateways, you can
...
+* Fix: Recount earning/sales tool for all downloads failing properly count.
+* Fix: Multiple instances of cart widget causes conflicts when adding or removing items.
+* Fix: API Key list table is unreadable on mobile.
+* Fix: Custom icons via edd_accepted_payment_icons are unwrapped.
+* Fix: Fees for specific download and price IDs do not check if that item or price ID is in the cart.
+* Fix: EDD API returning version 1 by default.
+* Fix: Change uses of jQuery .toggle() to use .show() and .hide() to fix bad checkbox states.
+* Fix: Correct translation for 'State / Province'.
+* Fix: Correct a variable type issue in the HTML_Elements::select defaults.
+* New: WP CLI: Customers command allows searching by user_id.
```
Comment (chriscct7) [inline readme.txt]: I thought we always listed new stuff first?

Comment (chriscct7) [inline readme.txt]: used to be 
new
enhancements
tweak
fix

for order

File: readme.txt [hunk 3]
```diff
@@ -187,6 +187,43 @@ Yes, through the addition of one or more of the add-on payment gateways, you can
...
+* New: Support iOS app one-click configuration.
+* New: Add hook after cart quantities are updated.
+* New: Add tooltip to "Display during checkout" tax option.
+* New: Allow exporting the file download history of a specific product.
+* New: Add action after discount is added to the cart.
+* New: Add action hook called "edd_payment_saved" for when EDD_Payment::save() is run.
+* New: Add "author" paramater support to [downloads] shortcode with support for username or ID.
+* Tweak: Correct branding of necessary assets.
+* Tweak: Block self-pings in tracking checkins.
+* Tweak: Update author and author URI to Easy Digital Downloads.
```
Comment (chriscct7) [inline readme.txt]: should add in plugin header. people might read this and think we're changing permalink slugs on their site

File: readme.txt [hunk 4]
```diff
@@ -222,7 +222,7 @@ Yes, through the addition of one or more of the add-on payment gateways, you can
 * Fix: EDD API was always returning version 1 by default.
 * Fix: Changed uses of jQuery .toggle() to use .show() and .hide() to fix bad checkbox states.
 * Fix: Corrected translation for 'State / Province'.
-* Fix: Corrected a variable type issue in the HTML_Elements::select defaults.
+* Fix: Corrected a variable type issue in the HwTML_Elements::select defaults.
```
Comment (chriscct7) [inline readme.txt]: Typo

**Comment by pippinsplugins** [comment]
Looks good.

**Pull request merged**